src/Native/Unix/System.Net.Security.Native/pal_gssapi.c

```diff
@@ -144,25 +144,27 @@ NetSecurityNative_ImportUserName(uint32_t* minorStatus, char* inputName, uint32_
     return gss_import_name(minorStatus, &inputNameBuffer, GSS_C_NT_USER_NAME, outputName);
 }
 
-uint32_t NetSecurityNative_ImportPrincipalName(uint32_t* minorStatus,
+uint32_t NetSecurityNative_ImportTargetName(uint32_t* minorStatus,
                                                char* inputName,
                                                uint32_t inputNameLen,
+                                               uint32_t isNtlmTarget,
                                                GssName** outputName)
```

Comment:
Nit: the indenting was previously aligned

Reply:
Fixed